app/components/data_protection_confirmation_banner_component.rb

```diff
@@ -13,13 +13,16 @@ def initialize(user:, organisation: nil)
   def display_banner?
     return false if user.support? && organisation.blank?
     return true if org_without_dpo?
+    return false if !org_or_user_org.holds_own_stock? && org_or_user_org.stock_owners.empty? && org_or_user_org.absorbed_organisations.empty?
```

Comment:
Need this cause in this case we should be displaying missing stock owners component